TTS/tts/utils/speakers.py

```diff
@@ -154,15 +155,21 @@ def save_d_vectors_to_file(self, file_path: str) -> None:
         """
         self._save_json(file_path, self.d_vectors)
 
-    def set_d_vectors_from_file(self, file_path: str) -> None:
+    def set_d_vectors_from_file(self, file_path: str, data: List = None) -> None:
```

Comment:
Function name already expects file_path. It doesn't make sense to add `data`. Then create a new function if it is needed.

Reply:
done